src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+//use props for functional components. .this for classes
+//cant use .state for functional 
+
+    const TodoForm = props => {
+        return (
+        <form onSubmit={(e) => props.handleChanges(e)} name='task'>       
+            <input onChange={props.onInputChange}
+            placeholder="Type here"
+            value={props.identity}
+            />
```

Comment:
if you add the word required to the end of your input.. your browser will prevent an empty task from being  sent through with an alert that says "must be filled" :)

ex:
 <input onChange={props.onInputChange}
            placeholder="Type here"
            value={props.identity}
            required
            />